store/types/utils.go

```diff
@@ -82,9 +82,8 @@ func PrefixEndBytes(prefix []byte) []byte {
 
 // InclusiveEndBytes returns the []byte that would end a
 // range query such that the input would be included
-func InclusiveEndBytes(inclusiveBytes []byte) (exclusiveBytes []byte) {
-	exclusiveBytes = append(inclusiveBytes, byte(0x00))
-	return exclusiveBytes
```

Comment:
I would recommend reverting this change, even though it's more verbose, I find the variable names helpful in understanding this functionality

Reply:
would it be better to better to comment instead of reverting?

Reply:
I think the variables would be clearer than a comment. But this is minor for me